src/elm/Session/LoggedIn.elm

```diff
@@ -607,6 +715,7 @@ viewMainMenu page model =
                 , ( activeClass, isActive page Route.Dashboard )
                 ]
             , Route.href Route.Dashboard
+            , onClick closeClaimWithPhoto
```

Comment:
Claim with photo dialog should be closed when the menu link is clicked. Didn't figure out how to it better...

Reply:
maybe we can do how we do when opening the profile menu, any outside click renders the whole modal to close, thoughts?

Reply:
I posted the reply [here](https://github.com/cambiatus/frontend/pull/476#issuecomment-778092083). Sorry for the mess, the comments didn't work here for some reason when I was replying 🤷🏻‍♂️

Reply:
Oh I see... it looks weird in the desktop version, but I got you!